cli/cmd/org/upload_logo.go

```diff
@@ -16,11 +16,12 @@ import (
 func UploadLogoCmd(ch *cmdutil.Helper) *cobra.Command {
 	var path string
 	var remove bool
+	var dark bool
 
 	cmd := &cobra.Command{
-		Use:   "upload-logo [<org-name> [<path-to-image>]]",
```

Comment:
I think the brackets should stay - for the org, it defaults to your default org, and for the path it's not required for `--remove`. Just for correctness – I realize this isn't super user friendly, but I guess it's kind of a rare command.